core/templates/pages/story-editor-page/story-preview-tab/story-preview-tab.component.html

```diff
@@ -1,23 +1,23 @@
 <div class="parent-container">
   <div class="oppia-story-viewer-title">
-    <[$ctrl.story.getTitle()]>
+    {{ story.getTitle() }}
   </div>
   <div class="oppia-story-viewer-description">
-    <[$ctrl.story.getDescription()]>
+    {{ story.getDescription() }}
   </div>
   <div class="oppia-story-header-text">
     <strong>
-      1 out of <[$ctrl.nodes.length]> Chapters Completed
+      1 out of {{ nodes?.length }} Chapters Completed
```

Comment:
@mridul-netizen  is `?` required here?

Reply:
@arpit1912  Angular was attempting to resolve `nodes.length` before it is resolved from our component file.This was giving errors in console like `cannot read property length of undefined`,so I used this "safe navigation operator" to avoid rendering content,until data is retrieved.https://stackoverflow.com/questions/46115643/http-get-response-updates-html-but-giving-undefined-error-in-console

Reply:
Can we add a comment about this in the code? Additionally, an entry in the Angular migration wiki might be good too.

Reply:
done

Reply:
done